src/main/kotlin/com/kotlinfarsi/Main.kt

```diff
@@ -1,33 +1,39 @@
 package com.kotlinfarsi
 
 import java.io.File
+import kotlin.system.measureTimeMillis
 
 class Main
 
 fun main() {
+    val mainInputDir = Main::class.java.getResource("/website").file
+    val finalOutputDir = convertToOutput(mainInputDir)
+    println("Main Input: $mainInputDir")
+    println("Final Output: $finalOutputDir")
+
+    val measureTime = measureTimeMillis {
+        // remove old output
+        File(finalOutputDir).takeIf { it.exists() }
+            ?.run { listFiles()?.forEach { it.deleteRecursively() } }
```

Comment:
delete recursively jaleb bood! nemidoonestam :D

Reply:
It's cuter ^__^
https://github.com/KotlinFarsi/kotlinfarsi-markdown-modifier/commit/365fb2e8f46c46b4e57a7d575f8592010ad0749b